htdocs/web_portal/components/Draw_Components/menu.php

```diff
@@ -100,10 +100,10 @@ function add_menu_item($menu_item)
     }
     if ($show == "all" or ($show=="write_enabled" and (!$portalIsReadOnly or $userisadmin)) or (($show == "admin") and ($userisadmin))){
         $html .=    	"<li class=\"Menu_Item\">".
-                        "<a href=\"".$link."\"><span class=\"menu_link\">".
+                        "<a href=\"".htmlspecialchars($link)."\"><span class=\"menu_link\">".
         $name."</span></a></li>";
```

Comment:
I've just noticed that the menu item names also contain ampersands and so also need escaping, I will add another commit.